README.md

```diff
@@ -39,58 +43,615 @@ Use a `<script>` tag to reference the package. Using this method will register a
 </html>
 ```
 
-If you need a specific version of Etvas Automat (and you most likely will), you can use it instead of `latest`:
+If you need a specific version of Etvas Automat (and you most likely will), you can use the version number (for example `1.3.0`) instead of `latest`:
 
 ```
   <script src="https://unpkg.com/@etvas/etvas-automat@1.3.0"></script>
 ```
 
 You can read more about resolving a version (major, minor and so on) by consulting the documentation at [unpkg.com](https://unpkg.com).
 
-In order to use Etvas Automat features, you need to initialize it. You can find the code fragment in your account on [Partners Portal](https://partners.helloetvas.com), which you must copy and paste in your HTML
+In order to use Etvas Automat features, you need to initialize it. You can find the code fragment in your account on [Partners Portal](https://partners.helloetvas.com), which you must copy and paste in your HTML.
+
+#### 2. When using a package manager
 
-#### The destination
+Etvas Automat is ready to be used with `npm` or `yarn`. The installation is straightforward:
 
-As this field is available for all our display blocks, it is worth to note what you can specify for the `destination` parameter.
+```
+$ npm install @etvas/etvas-automat`
+```
 
-You can specify an ID of a HTML element which is present in your DOM:
+or, if you prefer yarn:
 
 ```
-etvasAutomat.showProductCard('#my-placeholder-div', options)
+$ yarn add @etvas/etvas-automat
 ```
 
-Also, you can directly specify the JS DOM element:
+You can use the package as you normally use any other package. If you are using the CommonJS syntax, you can require the package like this:
 
 ```
-const el = document.getElementById('my-placeholder-div')
-etvasAutomat.showProductCard(el, options)
+const etvasAutomat = require("@etvas/etvas-automat");
+
+etvasAutomat.initialize({ ... });
+```
+
+But, as this is a Front-End SDK, you might use the `import` syntax:
+
+```
+import automat from "@etvas/etvas-automat";
+
+automat.initialize({ ... });
 ```
 
-Although not recommended, if you need to display it in multiple div's, identified by a common class, you can use:
+## Initializing the library
+
+The library needs some configuration in order to work and communicate properly with our backend services. We do that by calling a function named `initialize`. You will want to call this function as early as possible in your code, making sure it is the first one called:
+
+```
+automat.initialize({
+  etvasURL: "https://appname.etvasapps.com",
```

Comment:
I do not get what url is this ? it seems like an app url but which app ?

Reply:
ohh I've read that is the customer portal url .. maybe change the domain to helloetvas then ?